src/legacy/core_plugins/metrics/server/lib/vis_data/response_processors/annotations/index.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { flow } from 'lodash';
+import { makeFilter, annotationFilter } from '../../helpers/annotations';
```

Comment:
I don't understand why we should import **makeFilter** and **annotationFilter** from one file.

What about: 
1) makeFilter move to a separate file or make it private. 
2) in helpers/annotation do: 
`export const annotationFilter = makeFilter((annotations, filterValue) => annotations.filter(({ key }) => key <==filterValue));`

Reply:
Make sense. Done